app/javascript/packages/document-capture/components/document-capture.tsx

```diff
@@ -94,17 +93,11 @@ function DocumentCapture({ isAsyncForm = false, onStepChange = () => {} }: Docum
       field: error.field,
       error,
     }));
-  } else if (submissionError instanceof BackgroundEncryptedUploadError) {
-    initialActiveErrors = [{ field: submissionError.baseField, error: submissionError }];
   }
 
   let initialValues;
   if (submissionError && formValues) {
     initialValues = formValues;
-
-    if (submissionError instanceof BackgroundEncryptedUploadError) {
-      initialValues = except(initialValues, ...submissionError.fields);
```

Comment:
I think `except` is probably unused now.

Reply:
Per slack conversation, will remove.

Reply:
Removed.